### PR TITLE
Import utilty from my oss

### PR DESCRIPTION
**Pull request by nao1215**

### CODE REVIEW

File: file/file.go [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+	if runtime.GOOS == "windows" {
+		return strings.HasSuffix(path, ".exe")
+	}
+	stat, err := os.Stat(path)
+	return (err == nil) && ((stat.Mode() & Executable) != 0)
+}
+
+// IsHiddenFile reports whether the path exists and is included hidden file.
+func IsHiddenFile(filePath string) bool {
+	_, file := path.Split(filePath)
```
Comment (shogo82148) [inline file/file.go]: use https://pkg.go.dev/path/filepath#Split
`filePath` is a target operating system-defined file path.

```suggestion
	_, file := filepath.Split(filePath)
```

Comment (nao1215) [reply file/file.go]: Thank you for your review, @shogo82148 
I fixed it, see [8357b56](https://github.com/nao1215/gorky/pull/1/commits/8357b560a35d44738f4d752ceadd7807addd8c2c)

File: file/file.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+
+	_, err = io.Copy(d, s)
+	if err != nil {
+		return err
+	}
+	return nil
```
Comment (shogo82148) [inline file/file.go]: `d.Close()` may flush the internal buffer, and the caller wants to know its failure.
So you need to check the error of `d.Close()`.

```suggestion
	return d.Close()
```

Comment (nao1215) [reply file/file.go]: （いや、マージさせてくれ…）

Comment (nao1215) [reply file/file.go]: I'm sleepy...
Thanks, thanks for your review, yeah.

Fix [8415937](https://github.com/nao1215/gorky/pull/1/commits/84159372feeb8fe0805d792e44e908d5c1632ebb)

**Pull request merged**